index.d.ts

```diff
@@ -182,7 +183,7 @@ export enum HealthMetricType {
   MinWorkoutHR = "MinWorkoutHR",
   MaxWorkoutHR = "MaxWorkoutHR",
   SleepDuration = "SleepDuration",
-  SleepDurationInbed = "SleepDurationInbed",
+  SleepDurationAwake = "SleepDurationAwake",
```

Comment:
```suggestion
  SleepDurationInbed = "SleepDurationInbed",
  SleepDurationAwake = "SleepDurationAwake",
```

Keep both please, otherwise it'll break the Point App.
Just mark Inbed as deprecated but please do not delete it

Reply:
@AnnyFigueira done

Reply:
`SleepDurationInbed` was removed (not deprecated) from the Swift SDK, so I'm not sure if this will still break the Point App.

We can change the mapping to `SleepDurationInbed = "SleepDurationAwake"` and then deprecate it in favor of `SleepDurationAwake`

Reply:
SleepDurationInbed shouldn't have been removed from the Swift SDK, are you sure we removed it? I thought we only marked it as deprecated

Reply:
I can see it's still here https://github.com/agencyenterprise/point-sdk-ios/blob/main/PointSDK/API/ApolloAPI.swift
But not mapped in the data model. 
So... should we change the day the Point App calls this metric? We can do it there if this was really removed from the SDK

Reply:
Mapping the enum as I suggested won't work for Point app?